Core/OxidEeEvents.php

```diff
@@ -201,6 +201,14 @@ public static function extendPaymentMethodTable()
             " ADD COLUMN `WDOXIDEE_LOGOVARIANT` enum('standard', 'descriptive')";
         self::_addColumnIfNotExists(self::PAYMENT_TABLE, 'WDOXIDEE_LOGOVARIANT', $sQueryAddLogoVariant);
 
+        $sQueryAddTrusedShop = "ALTER TABLE " . self::PAYMENT_TABLE .
```

Comment:
I think this is a good use case for the new metadata functionality introduced in [WDPD-168](https://github.com/wirecard/oxid-ee/pull/110). Using this, you don't have to create new columns for this payment-method-specific setting.

Reply:
:+1: